cli/lib/exec/spawn.js

```diff
@@ -251,8 +251,12 @@ module.exports = {
             // the stderr logs unless we've explicitly
             // enabled the electron debug logging
             if (!debugElectron.enabled) {
+              debugElectron(str)
```

Comment:
This doesn't make sense - "if the Electron debugger is disabled, then log this using the Electron debugger"?